src/tezos_rpc/fetch_block_operations.ml

```diff
@@ -0,0 +1,176 @@
+open Helpers
+open Crypto
+open Tezos
+open Http
+
+type parameters = {
+  entrypoint : string;
+  value : Michelson.t;
+}
+type internal_operation =
+  | Internal_transaction     of {
+      sender : Address.t;
+      destination : Address.t;
+      parameters : parameters option;
+    }
+  | Internal_non_transaction
+
+type operation_status =
+  (* if one of the internal operations the status will be "backtracked" *)
+  | Applied
+  | Other
```

Comment:
I don't understand this comment, why is "backtracked" not one of the variants if it's a possible status?

Reply:
This PR intentionally tries to over simplify the format, there is like 5 or 6 status, in the future we can improve this, maybe by removing it from Tezos directly.